homeassistant/components/alexa/smart_home.py

```diff
@@ -34,6 +36,19 @@
     TEMP_CELSIUS: 'CELSIUS',
 }
 
+API_THERMOSTAT_MODES = {
+    climate.STATE_HEAT: 'HEAT',
+    climate.STATE_COOL: 'COOL',
+    climate.STATE_AUTO: 'AUTO',
+    climate.STATE_ECO: 'ECO',
+    STATE_ON: 'AUTO',
+    STATE_OFF: 'OFF',
+    STATE_IDLE: 'OFF',
+    climate.STATE_FAN_ONLY: 'OFF',
+    climate.STATE_DRY: 'OFF',
```

Comment:
Damn there is so much shit in the climate component. One of these days I am going on a great purge of these states.